frictionless/helpers.py

```diff
@@ -305,6 +305,15 @@ def unzip_descriptor(path, innerpath):
             tempdir = tempfile.mkdtemp()
             zip.extractall(tempdir)
             atexit.register(shutil.rmtree, tempdir)
+            if innerpath != "":
+                pass
+            elif os.path.isfile(os.path.join(tempdir, "datapackage.json")):
+                innerpath = "datapackage.json"
+            elif os.path.isfile(os.path.join(tempdir, "datapackage.yaml")):
+                innerpath = "datapackage.yaml"
+            else:
+                # maybe an error
+                innerpath = "datapackage.json"
```

Comment:
```suggestion
            if innerpath == "":
                innerpath = "datapackage.json"
                extensions = ("json", "yaml", "yml")
                default_names = (f"datapackage.{ext}" for ext in extensions)
                for name in default_names:
                    if os.path.isfile(os.path.join(tempdir, name)):
                        innerpath = name
                        break
```

How about we change it like this to avoid some repetition? This also passes the test and has the same effect (plus adds a heuristic for also finding `.yml`).

Reply:
nice!!